CONTRIBUTING.md

```diff
@@ -66,14 +64,23 @@ export USERNAME={userName}
 export PASSWORD={password}
 ```
 
-After setting up the environment variables, run this script to copy the configuration into the JSON configuration files:
+For Windows, please set the following environment variables - ISSUER, CLIENT_ID, USER_NAME, PASSWORD
+
+> **NOTE:** Windows has USERNAME as a built-in system variable, hence set the USER_NAME environment variable for testing.
+
+After setting up the environment variables, run this node script to copy the configuration into the JSON configuration files:
 
-```bash
-sh scripts/setup-env.sh
+```
+node scripts/setup-env.js
```

Comment:
Been meaning to mention this: this script seems to be idempotent, why not just call it automatically as part of the `npm test` command?

Reply:
Makes sense. Will update.